### PR TITLE
document object_mkk

### PR DESCRIPTION
**Pull request by MegaIDK**
really werid dls on this one

### CODE REVIEW

File: src/overlays/actors/ovl_En_Door/z_en_door.c [hunk 1]
```diff
@@ -379,7 +379,7 @@ static Gfx* D_808679A4[14][2] = {
     { gMilkBarDoorDL, gMilkBarDoorDL },
     { gMusicBoxHouseDoorDL, gMusicBoxHouseDoorDL },
     { gPiratesFortressDoorDL, gPiratesFortressDoorDL },
-    { object_mkk_DL_000310, object_mkk_DL_000310 },
+    { gWhiteBoeEyesDL, gWhiteBoeEyesDL },
```
Comment (isghj5) [inline src/overlays/actors/ovl_En_Door/z_en_door.c:382]: wat. Door?

Comment (hensldm) [reply src/overlays/actors/ovl_En_Door/z_en_door.c:382]: Ahh, that looks like a mistake. They should be the DLs from kinsta2, not mkk.

Comment (MegaIDK) [reply src/overlays/actors/ovl_En_Door/z_en_door.c:382]: ikr??? no idea

Comment (MegaIDK) [reply src/overlays/actors/ovl_En_Door/z_en_door.c:382]: > Ahh, that looks like a mistake. They should be the DLs from kinsta2, not mkk.

ohh ic so do i just leave em like that

Comment (hensldm) [reply src/overlays/actors/ovl_En_Door/z_en_door.c:382]: If you wouldn't mind changing them to use the kinsta2 dls that would be better. Would need to change the header include to the kinsta2 object, not the mkk one.

Comment (MegaIDK) [reply src/overlays/actors/ovl_En_Door/z_en_door.c:382]: might take the opportunity to do the remaining door assets as well then but sure

**Review by hensldm — APPROVED** [review]
LGTM (since you are doing the door dls in a different PR, don't worry about it here)

**Review by AngheloAlf — APPROVED** [review]

File: assets/xml/objects/object_mkk.xml [hunk 2]
```diff
@@ -1,15 +1,16 @@
 ﻿<Root>
+    <!--Object for Black and White Boes-->
```
Comment (engineer124) [inline assets/xml/objects/object_mkk.xml:2]:
```suggestion
    <!-- Object for Black and White Boes -->
```

**Review by engineer124 — APPROVED** [review]

**Pull request merged**